internal/v2/tls_config_store/example_cert_key/client_cert.pem

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I want to move this example_cert_key directory to be a subdirectory of internal/v2, since it will be used in testing of future libraries inside internal/v2. I was running into an error, similar to this discussion:https://github.com/golang/go/issues/46056. I wasn't able to resolve it during this PR, which is why I left example_cert_key inside the tls_config_store directory. Looking back, it may be with how I was formatting the string paths in the BUILD file. I'll try to move this folder one level up in the next PR.

Reply:
From this link : https://go.googlesource.com/proposal/+/master/design/draft-embed.md#:~:text=An%20embed.Files,no%20embedded%20files, it seems to imply that data files cannot be in parent directories. I run into the following error when I put the data files in the interna/v2 directory:

"missing input file '//internal/v2/tls_config_store:s2a-go/internal/v2/example_cert_key/client_cert.pem' "

Reply:
Interesting. Let's leave it in whichever directory works for now, and add a TODO to come back to this later.

Rationale: At the end of the day when the full S2A-Go client is complete, I think these test files will only be used by the fake S2A implementation. So let's defer figuring this out exactly where to put these until we know exactly where they're needed.

Reply:
Will do, thanks for the rationale!